test/image/mocks/gl2d_selectedpoints.json

```diff
@@ -10,5 +10,24 @@
       "opacity": [0.9, 0.8, 0.7, 1, 0.6, 0.8]
     },
     "selectedpoints": [1, 4, 2]
-  }]
+  }, {
+    "name": "array marker opacity + set unselected.marker.opacity",
+    "type": "scattergl",
+    "mode": "markers",
+    "x": [1, 2, 3, 4, 5, 6],
+    "y": [3, 5, 4, 6, 7, 9],
+    "marker": {
+      "size": 20,
+      "opacity": [0.9, 0.8, 0.7, 1, 0.6, 0.8]
+    },
+    "unselected": {
+      "marker": {
+        "opacity": 0
```

Comment:
Before this commit, this gave:

![image](https://user-images.githubusercontent.com/6675409/37985848-e3513184-31c7-11e8-8fd4-ea424729244b.png)

that is `unselected.marker.opacity` was overrode.